packages/trader/src/Stores/Modules/Portfolio/portfolio-store.js

```diff
@@ -133,7 +133,10 @@ export default class PortfolioStore extends BaseStore {
 
     updateContractReplayStore(response) {
         const contract_replay = this.root_store.modules.contract_replay;
-        if (contract_replay.contract_id === response.proposal_open_contract?.contract_id) {
+        if (
+            contract_replay.contract_id === response.proposal_open_contract?.contract_id ||
+            contract_replay.contract_id
```

Comment:
This will update `contract_replay` store for multiple `proposal_open_contract` subscription messages received.
Is there any way to get `contract_id` from `proposal_open_contract` error response?

Reply:
@balakrishna-binary sure, as we discussed, I just fix this.